joint_trajectory_controller/test/joint_trajectory_controller_test.cpp

```diff
@@ -1266,7 +1277,7 @@ TEST_F(JointTrajectoryControllerTest, goalToleranceViolation)
   // Make robot respond with a delay
   {
     std_msgs::Float64 smoothing;
-    smoothing.data = 0.95;
+    smoothing.data = 0.98;
```

Comment:
Why did you chnage this?

Reply:
This makes the movement faster.

Reply:
A smoothing value of 0 corresponds to direct control whereas a value of 1 makes the robot not respond at all. Therefore, we movement is delayed more. "Slower" was the wrong term, I guess.

Reply:
Ah, you are right, my bad!